tests/test_character.py

```diff
@@ -1,8 +1,8 @@
 import unittest
-import tests.utils
+#import tests.utils
```

Comment:
@sergebruni This line probably is conflicting with the latest master. Just delete it and update your PR ;)